packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts

```diff
@@ -236,6 +236,18 @@ class DataStorageAuthWithTriggerTestProject extends TestProjectBase {
       backendId,
       'AWS::IAM::Role'
     );
+
+    // ensure typed shim files are generated by checking for node16Function's typed shim
+    const typedShimStats = await fs.stat(
+      path.join(
+        this.projectDirPath,
+        '.amplify',
+        'function-env',
+        'node16Function.ts'
```

Comment:
nit: since node16 is going away soon, can we do this check on the other lambda so we don't have to update this when removing node 16?

Reply:
Sure, I'll do this for one of the s3 trigger lambdas since `defaultNodeFunction` typed shim is guaranteed to be there